README.md

```diff
@@ -1,20 +1,13 @@
-### A Dart 2D physics engine
-
-[![Build Status](https://travis-ci.org/google/box2d.dart.svg?branch=master)](https://travis-ci.org/google/box2d.dart)
-[![Coverage Status](https://coveralls.io/repos/google/box2d.dart/badge.svg?branch=master)](https://coveralls.io/r/google/box2d.dart)
-
-*This is a Dart port of Java's Box2D libraries.*
-
-*The Java Box2D library was originally written by Daniel Murphy (status
-February 2015).*
-
-```
-benchmark/bench2d.dart   Benchmark used by Joel Webber
-example/index.html       Browser demos as written by Dominic Hamon
-lib/                     Port of Daniel Murphy's jbox2d
-```
-
-First Dart port of an earlier version of Box2D library was
-done by Dominic Hamon. We are grateful for his work.
-
-__*Not an official Google project*__
+## Forge2D - A Dart port of the Box2D physics engine
```

Comment:
nit: first level should be `#` right?

nit: add our badegs (pub badge, build badge and discord badge)

Reply:
Fixed, just have to get a logo for the shield. :)

Reply:
cc @renancaraujo